fips.Dockerfile

```diff
@@ -0,0 +1,134 @@
+# check=skip=RedundantTargetPlatform
+# setup build image
+FROM --platform=$TARGETPLATFORM mcr.microsoft.com/oss/go/microsoft/golang:1.24.2-fips-bullseye@sha256:28ab4742d3b5feb0b3c7450629b4e105128d8709dd6ca22898472ab302140c37 AS operator-build
+
+ENV GOEXPERIMENT=systemcrypto
+
+WORKDIR /app
+
+ARG DEBUG_TOOLS
+RUN if [ "$DEBUG_TOOLS" = "true" ]; then \
+      GOBIN=/app/build/_output/bin go install github.com/go-delve/delve/cmd/dlv@latest; \
+    fi
+
+COPY go.mod go.sum ./
+RUN go mod download -x
+
+COPY pkg ./pkg
+COPY cmd ./cmd
+
+ARG GO_LINKER_ARGS
+ARG GO_BUILD_TAGS
+ARG TARGETARCH
+ARG TARGETOS
+
+RUN --mount=type=cache,target="/root/.cache/go-build" \
+    --mount=type=cache,target="/go/pkg" \
+    CGO_ENABLED=1 GOFIPS=1 GOOS=$TARGETOS GOARCH=$TARGETARCH \
+    go build -tags "${GO_BUILD_TAGS}" -trimpath -ldflags="${GO_LINKER_ARGS}" \
+    -o ./build/_output/bin/dynatrace-operator ./cmd/
+
+# ---------------- Install Packages in Final Image -----------------------
```

Comment:
```suggestion
```
i don't see any benefits, and i prefer to be consistent with original Dockerfile, hope one day it will be same Dockerfile.